index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>Document</title>
```

Comment:
WHAT THE HELL IS GOING ON!?!?!? I TOLD YOU TO TITLE IT, COOKIE MONSTER, NOT LEAVE IT AT DOCUMENT!

Reply:
okay will push up changes right away